mne/viz/tests/test_3d.py

```diff
@@ -206,7 +209,7 @@ def test_plot_alignment(tmpdir, renderer, mixed_fwd_cov_evoked):
                   subject='fsaverage', subjects_dir=subjects_dir,
                   src=sample_src)
     sample_src.plot(subjects_dir=subjects_dir, head=True, skull=True,
-                    brain='white')
+                    brain='white', trans=trans_fname)
```

Comment:
This should not be necessary and should not raise an error. You should only need `trans` if it's needed to get from the original coordinate frame (here, MRI, as that's where source spaces live) to the destination plotting frame (here, still MRI).